docs/en/latest/concepts/apisix_route.md

```diff
@@ -262,6 +262,46 @@ spec:
           secretRef: echo
 ```
 
+## Config with secretRef configuring nested fields
```

Comment:
Could you come up with a better title? Struggle to understand what this is trying to say

Reply:
@kayx23 Please check if current title is fine. I made the change/